### DISCUSSION TITLE
v1.5 on https://hub.docker.com/r/gramineproject/gramine

### DISCUSSION THREAD

**Discussion by haraldh** [gramineproject/gramine#1449] [General]
I am still missing a v1.5 version. Might be a job for a github actions workflow triggered by release tags :)

**Comment by dimakuv** [comment]
We're building this Docker image manually for every release. The work is currently being done. Please wait for another 1-2 days.

**Comment by haraldh** [comment]
Thank you!

**Comment by haraldh** [comment]
btw, where is the Dockerfile stored?

**Comment by haraldh** [comment]
https://github.com/gramineproject/gramine/blob/master/packaging/docker/Dockerfile

**Reply by dimakuv** [reply]
Yes, exactly.
